test/functional/rpc_mempool_entry_fee_fields_deprecation.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021 The Bitcoin Core developers
+# Distributed under the MIT software license, see the accompanying
+# file COPYING or http://www.opensource.org/licenses/mit-license.php.
+"""Test deprecation of fee fields from top level mempool entry object"""
+
+from test_framework.wallet import MiniWallet
+from test_framework.test_framework import BitcoinTestFramework
+from test_framework.util import assert_equal
+from test_framework.blocktools import COIN
+
+
+class MempoolFeeFieldsDeprecationTest(BitcoinTestFramework):
+    def set_test_params(self):
+        self.num_nodes = 2
+        self.extra_args = [[], ["-deprecatedrpc=fees"]]
+
+    def run_test(self):
+        # we get spendable outputs from the premined chain starting
+        # at block 76. see BitcoinTestFramework._initialize_chain() for details
+        self.wallet = MiniWallet(self.nodes[0])
+        self.wallet.scan_blocks(start=76, num=1)
+
+        # we create the tx on the first node and wait until it syncs to node_deprecated
+        # thus, any differences must be coming from getmempoolentry or getrawmempool
+        txid = self.wallet.send_self_transfer(from_node=self.nodes[0])['txid']
+        self.sync_mempools(nodes=self.nodes)
+
+        # note: we are only testing getmempoolentry even though
+        # getrawmempool(verbose=True) also returns mempool entries
+        # this is because getmempool entry and getrawmempool(verbose=True)
+        # both call the same underlying entryToJSON function so testing
+        # getmempoolentry is sufficient
```

Comment:
You could also add the assertions to a helper function, and call that helper function with results you get from each RPC